src/router/index.js

```diff
@@ -29,6 +34,18 @@ const routes = [
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
     component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
+  },
+  {
+    path: '/signin',
+    name: 'SignIn',
+    component: SignIn,
+    beforeEnter: (to, from, next) => {
+      if (userService.isLoggedIn()) {
+        next({ name: 'Home' })
+      } else {
+        next()
+      }
+    }
```

Comment:
Könntet ihr hier nicht auch die schon bestehende **_guard_** Funktion benutzen?
```suggestion
    beforeEnter: guard,
```

Reply:
Nein, wir möchten hier expliziert auf die Home Seite. :)